### DISCUSSION TITLE
Welcome to xlDuckDb Discussions!

### DISCUSSION THREAD

**Discussion by RusselWebber** [RusselWebber/xlDuckDb#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by djouallah** [comment]
man, this is awesome !!!!

**Reply by RusselWebber** [reply]
Thanks very much! Hope you enjoy using xlDuckDb.

**Comment by JohnBianQi** [comment]
Thank you so much for creating such a great add-in for EXCEL users!
Any plan to release a new version? The latest DuckDB is '1.5.0'.

**Reply by RusselWebber** [reply]
Thank you for the kind words.
I have just released a new version targetting DuckDb 1.5.0
Note that the new VARIANT type is not yet supported.

**Reply by JohnBianQi** [reply]
Well noted! Many thanks for your prompt response.
